The Elasticsearch container won't start, uncaught Exception

**solas version**:  In quay the elasticsearch-container is latest.  Note: There is not an older version available in quay.

Please provide a link to the repo you are trying to create.
I tried to install [this chart](https://github.com/samsung-cnct/chart-elasticsearch/).

**Environment**:  Testing with Kubernetes 1.8.2, and the latest chart-elasticsearch


**What happened**:
I installed the chart: 
    helm registry install quay.io/samsung_cnct/elasticsearch-chart
I looked at the pods and they were in state 'CrashLoopBackOff'
    kubectl get pods
Looked at the pod log:
```1: Before ulimit
64
2. After ulimit
unlimited
[2017-11-13T22:23:54,037][INFO ][o.e.n.Node               ] [elasticsearch-master-0] initializing ...
[2017-11-13T22:23:54,143][WARN ][o.e.b.ElasticsearchUncaughtExceptionHandler] [elasticsearch-master-0] uncaught exception in thread [main]
org.elasticsearch.bootstrap.StartupException: java.lang.IllegalStateException: Failed to create node environment
	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:127) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.bootstrap.Elasticsearch.execute(Elasticsearch.java:114) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:67) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:122) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.cli.Command.main(Command.java:88) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:91) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:84) ~[elasticsearch-5.5.2.jar:5.5.2]
Caused by: java.lang.IllegalStateException: Failed to create node environment
	at org.elasticsearch.node.Node.<init>(Node.java:267) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.node.Node.<init>(Node.java:244) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:232) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:232) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:351) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:123) ~[elasticsearch-5.5.2.jar:5.5.2]
	... 6 more
Caused by: java.nio.file.AccessDeniedException: /usr/share/elasticsearch/data/nodes
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:84) ~[?:?]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102) ~[?:?]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107) ~[?:?]
	at sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:384) ~[?:?]
	at java.nio.file.Files.createDirectory(Files.java:674) ~[?:1.8.0_141]
	at java.nio.file.Files.createAndCheckIsDirectory(Files.java:781) ~[?:1.8.0_141]
	at java.nio.file.Files.createDirectories(Files.java:767) ~[?:1.8.0_141]
	at org.elasticsearch.env.NodeEnvironment.<init>(NodeEnvironment.java:221) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.node.Node.<init>(Node.java:264) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.node.Node.<init>(Node.java:244) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:232) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:232) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:351) ~[elasticsearch-5.5.2.jar:5.5.2]
	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:123) ~[elasticsearch-5.5.2.jar:5.5.2]
```

**What you expected to happen**:
I expected 'k get pods' to show the Running state.

**How to reproduce it** (as minimally and precisely as possible):
Please see steps in 'What Happened' above.

**Anything else we need to know**:
This was working with the same version of kubernetes last Wednesday.  We have merged some changes since then:
https://github.com/samsung-cnct/chart-elasticsearch/pull/16
https://github.com/samsung-cnct/container-elasticsearch/pull/9

--- a/Dockerfile
+++ b/Dockerfile
@@ -23,4 +23,6 @@ ADD limits.conf /etc/security/limits.conf
 
 COPY custom-entrypoint.sh /custom-entrypoint.sh
 
+VOLUME ["/usr/share/elasticsearch"]
+
 ENTRYPOINT ["/custom-entrypoint.sh"]

